cityofaustin/atd-product
# 10722 about page

Fixes cityofaustin/atd-data-tech/issues/10722

Test steps:
[
Netlify link](https://deploy-preview-54--atd-product.netlify.app/about)

1. Navigate to About page.
2. Check how everything looks
3. Resize to a tablet breakpoint. "How we work" section should have 2 columns and team pics should have 3 columns, then resize to mobile breakpoint, "How we work" section should have 1 column and team pics should have 2 columns.
